SMDS_3/include/CGAL/IO/File_binary_mesh_3.h

```diff
@@ -46,7 +46,7 @@ save_binary_file(std::ostream& os,
                , bool binary = true)
 #endif
 {
-  typedef typename C3T3::Triangulation::Geom_traits::FT FT;
+  typedef typename C3T3::Geom_traits::FT FT;
```

Comment:
this will work in practice, but there is no `Geom_traits` in C3T3

Reply:
I have reverted that change. See c0cb990cb9aeccbd448135bd5f507f3e23146927.